frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/HPA _Topology_&_Sidebar.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Add HPA action and topology sidebar modifications
+	As a user, I want to add a HPA to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment
+    And user has added CPU resource limit
+    And user has added Memory resource limit
+    And user has a workload with HPA assigned to it
+
+@regression @manual
+Scenario: Add HPA Sidebar
```

Comment:
Can you please describe it  more what exactly you are trying to prove here? Also this is not a scenario of adding a HPA or Add HPA doesn't have any sidebar. The sidebar will be for workload and there will be section for HPA. Can you please update it accordingly.